beacon_node/beacon_chain/tests/store_tests.rs

```diff
@@ -299,7 +299,7 @@ async fn light_client_updates_test() {
         .unwrap()
         .unwrap();
 
-    let kzg = spec.deneb_fork_epoch.map(|_| KZG.clone());
+    let kzg = KZG_NO_PRECOMP.clone();
```

Comment:
same func here

not sure if using pre-comp here will have a perf impact, was this intentional?